ioctl/cmd/contract/contract.go

```diff
@@ -141,7 +141,13 @@ func packArguments(targetAbi *abi.ABI, targetMethod string, rowInput string) ([]
 	for _, param := range method.Inputs {
```

Comment:
i think this way would be more clear?
```go
if param.Name == "" {
   param.Name = "_"
}
```
before `rowArg, ok := rowArguments[param.Name]`

Reply:
OK